docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Linode Code of Conduct
+
+## Our Pledge
+
+In order to encourage an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of: 
+
+* age
+* body size
+* disability
+* ethnicity
+* gender identity and expression
+* level of experience
+* nationality
+* personal appearance
+* race
+* religion
+* sexual identity and orientation
+
+## Our Standards
+
+You can contribute to creating a positive environment in many ways. For example you can: 
+
+* use welcoming and inclusive language
+* be respectful of differing viewpoints and experiences
+* accept constructive criticism gracefully
+* focus on what is best for the community
+* show empathy towards other community members
+* be helpful and understanding
+
+You should not: 
+
+* use sexualized language or imagery
+* make unwelcome sexual advances
+* troll, and make insulting or derogatory comments
+* make personal or political attacks
+* harass others, in public or private
+* publish others' private information, such as a physical or electronic address, without explicit permission
+* engage in any other conduct which could reasonably be considered bullying or inappropriate in a professional setting
+
+## Our Responsibilities
+
+As project maintainers, we are responsible for clarifying the standards of acceptable behavior and we are expected to take appropriate and fair corrective action in response to any instances of unacceptable behaviour.
+
+We have the right and responsibility to remove, edit, or reject:
+
+* comments
+* commits
+* code
+* wiki edits
```

Comment:
No Wiki Consider removing

Reply:
@Guaris would we want to keep it as a "if ever we add a wiki, this applies there too," or would you rather we just re-edit when/if ever we add a wiki?